src/compose/ComposeBox.js

```diff
@@ -217,6 +220,54 @@ class ComposeBox extends PureComponent<Props, State> {
     }
   };
 
+  insertAttachment = async (attachments: DocumentPickerResponse[]) => {
+    this.setState(({ numUploading }) => ({
+      numUploading: numUploading + 1,
+    }));
+
+    const { _, auth } = this.props;
+
+    const placeholders: string[] = [];
+    const fileNames: string[] = [];
+
+    for (let i = 0; i < attachments.length; i++) {
+      const fileName = attachments[i].name ?? `${_('Attachment {num}', { num: i + 1 })}`;
```

Comment:
nit: this has the structure `` `${/* … some expression … */}` ``, right? Simpler to drop the backticks and the `${` / `}`, then.